lib/Config.js

```diff
@@ -86,58 +91,51 @@ class Config {
         }
 
         // Read config automatically
-        this._getLocationConfig();
         this._getConfig();
+        this._getLocationConfig();
+        this._configureBackends();
     }
 
     _getLocationConfig() {
         let locationConfig;
-        // check exist file
-        try {
-            fs.statSync(this.locationConfigPath);
-        } catch (err) {
-            if (err) {
-                // location config path is invalid
-                // legacy config format where only one config file
-                return;
+        if (this.restEndpoints) {
```

Comment:
I don't think this should be in a conditional anymore.  They must have the locationConfig